index.html

```diff
@@ -161,6 +163,27 @@ <h3 class="inner">skills</h3>
             </ul>
         </div>
     </div>
+</section>
+<section class="about-me">
+  <div class="formshapes"></div>
+<h1 class="cont">
+    Contact me
+  </h1>
+  <p class="contact-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+  <form action="https://formspree.io/f/maykjvbv" method="post" class="form">
+    <label for="name" >
+      <input type="name" name="user_name"class="input" placeholder=" Enter your name" maxlength="30" required >
+    </label>
+    <label for="email">
+      <input type="email" name="user_email"class="email"placeholder=" Enter your email id" required>
+    </label>
+    <textarea name="msg" id="message" cols="33" rows="10" placeholder=" Write your message here"  maxlength="500" required></textarea>
+    <button type="submit" value="submit" class="submit">Get it touch
+   
+    </button>
+    <img src="images/Bottom-line.svg" id="btm-lne">
+  </form>
```

Comment:
Good job! but as requested by the previous reviewer, please make sure to the design as much as possible.

- [ ] 1. Kindly use the same font size and color
- [ ] 2. Please try to enhance your design, for example, you can remove the border from the inputs and also add some padding to the inputs.
- [ ] 3. The bottom image doesn't appear, please make sure to fix it.

![image](https://user-images.githubusercontent.com/34779928/194126616-464f8b65-969a-471b-bfbb-8ecf0db0aa8e.png)

Reply:
fonts fize and font-style is fixed 
put the padding as per required
bottom image issue is fixed